### ISSUE TITLE
Buttons lack hover and focus states

### ISSUE THREAD

**Issue by carlosperez** [carlosperez/audit#1]
### Description
Buttons are not setup with `:hover` and `:focus` states when a user mouses over a button or focuses a button with the keyboard.

### Effect
Users do not know if something is enabled, or can be interacted with.

### Potential Solution
Add CSS rules for clear hover and focus states.

---

#### Screenshots
[Link to screenshot annotation tool](#)
